django/contrib/admin/options.py

```diff
@@ -555,7 +556,12 @@ def wrapper(*args, **kwargs):
             url(r'^add/$', wrap(self.add_view), name='%s_%s_add' % info),
             url(r'^(.+)/history/$', wrap(self.history_view), name='%s_%s_history' % info),
             url(r'^(.+)/delete/$', wrap(self.delete_view), name='%s_%s_delete' % info),
-            url(r'^(.+)/$', wrap(self.change_view), name='%s_%s_change' % info),
+            url(r'^(.+)/edit/$', wrap(self.change_view), name='%s_%s_change' % info),
```

Comment:
If we had to do it again, "edit" would probably make sense, but I think "change" makes sense given that terminology is used elsewhere "change view", "change permission", etc.